lisa/tests/base.py

```diff
@@ -903,13 +940,25 @@ def rtapp_profile(self):
         return self.get_rtapp_profile(self.plat_info)
 
     @property
-    def rtapp_tasks(self):
+    def rtapp_tasks_from_profile(self):
```

Comment:
You continue to provide this in case somebody wants to use it still, right? Do we need to?

Reply:
Removed, since this is also a quite dummy wrapper we can avoid.